### PR TITLE
WS1 - Static Recipe - notevenalex

### PR DESCRIPTION
**Pull request by notevenalex**
### Acceptance Criteria

- [ ] You must use good [semantics in HTML tags](https://es.semrush.com/blog/html-semantico/).
- [x] You have to import fonts from a CDN or create a font from 0 with [@font-face](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face).
- [x] Use [SASS preprocessor](https://sass-lang.com/documentation/style-rules/) or others ([Less vs Sass vs Stylus](https://stackshare.io/stackups/less-vs-sass-vs-stylus)).
- [ ] Use [variables](https://sass-lang.com/documentation/variables/) for style sheets without hard code.
- [x] You have to [reset the CSS](https://www.aluracursos.com/blog/reset-css-que-es-ejemplos-como-crear-y-utilizar) so that you can work with the HTML elements in a clean way.
- [ ] You have to use properties of [box model](https://web.dev/i18n/es/learn/css/box-model/) such as box-sizing, margin, padding, and border.
- [ ] You have to use CSS display layout properties:
  - [CSS Inline vs Inline-Block vs Block](https://www.samanthaming.com/pictorials/css-inline-vs-inlineblock-vs-block/)
  - [Grid & flexbox MDN](https://developer.mozilla.org/es/docs/Web/CSS/CSS_grid_layout/Relationship_of_grid_layout_with_other_layout_methods#grid_y_flexbox)
- [x] You have to use CSS [positioning](https://css-tricks.com/almanac/properties/p/position/) and [z-index](https://css-tricks.com/css-painting-order/) properties to place the overlapping elements.

### Demo
imagen

---
### Comentarios Taller:

- Te falta 4 criterio de aceptación:
**Nota:** `2.5 de 5.0`

> Te hacen falta:
  - Corregir comentarios sobre Semántica HTML
  - No usar variables quemadas
  - No se ven desarrollo de flex o grid, margin, paddin y z-index
> Te recomiendo que te apoyes en los canales de [#frontend-help](https://nodo-eafit.slack.com/archives/C046MM1ABGQ), con tus compañeros o Pedir [Tutoría Académicas](https://bit.ly/441DtaF)

### CODE REVIEW

**Comment by DanteCely** [comment]
Te falta implementar SASS 🙄

**Comment by notevenalex** [comment]
> Te falta implementar SASS 🙄

Listo

File: notevenalex/book-recipe/cooking-recipe.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <main>
+            <aside></aside>
+            <section></section>
+        <div class="background-recipe"></div>
+        <main class="cooking.recipe">
+            <header>
+                <img src="./img/Logo-mas-receta.png" alt="Receta de espárragos" />
+                <h1>Platos Calientes</h1>
+                <h2>Tosta de espárragos gratinados</h2>
+                <h3>con mayonesa</h3>
```
Comment (DanteCely) [inline notevenalex/book-recipe/cooking-recipe.html:24]: Se ve que hace parte del `<h2>`
Si quieres usar un salto de linea utiliza dentro del `<h2>` un `<br>`

File: notevenalex/book-recipe/cooking-recipe.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                        <span>30 min</span>
+                    </div>
+                    <div class="recipe-property">
+                        <i>Costo:</i>
+                        <span>Bajito</span>
+                    </div>
+                    <div class="recipe-property">
+                        <i>Nivel Chefistico:</i>
+                        <span>Sencillito</span>
+                    </div>
```
Comment (DanteCely) [inline notevenalex/book-recipe/cooking-recipe.html:52]: Utiliza una lista no ordenada para los `<div class="recipe-property-container">`
[HTML `<ul> `Tag](https://www.w3schools.com/tags/tag_ul.asp)

File: notevenalex/book-recipe/cooking-recipe.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                </p>
+                <p>
+                    Colocamos encima de cada rebanada tres espárragos y sobre ellos dos cucharadas de mayonesa.
+                </p>
+                <p>
+                    Añadimos un poco de queso mozzarella rallado.
+                </p>
+                <p>
+                    Introducimos la bandeja en el horno precalentado a 180° y gratinamos hasta que se dore la mayonesa con queso.
+                </p>
```
Comment (DanteCely) [inline notevenalex/book-recipe/cooking-recipe.html:81]: Utiliza una lista ordenada para los párrafos.
[HTML `<ol>` Tag](https://www.w3schools.com/tags/tag_ol.asp)

File: notevenalex/book-recipe/cooking-recipe.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                <p>
+                    Mientras los brotes jóvenes de los espárragos van creciendo dentro de la tierra éstos son de color blanquecino. 
+                </p>
+                <p>
+                    Pero cuando emergen del suelo y entran en contacto con la luz, adquieren una coloración verde, debido a que la misma, activa la función clorofílica.
+                </p>
+                <p>
+                    El espárrago blanco, por tanto, se recolecta tan pronto como emerge de la tierra, mientras que los verdes se espera a cortarlos cuando alcanzan una altura sobre el suelo de unos 20 cm.
+                </p>
```
Comment (DanteCely) [inline notevenalex/book-recipe/cooking-recipe.html:106]: Utiliza una lista no ordenada para los párrafos.
[HTML `<ul>` Tag](https://www.w3schools.com/tags/tag_ul.asp)

File: notevenalex/book-recipe/sass-folder/styles.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
+@import "../variables.scss";
+
+header{
+  h2 {background-color: "$red-1";
+    width: 40%;
+}
+}
+body {
+  position: relative;
+  background-color: "$beige-1";
```
Comment (DanteCely) [inline notevenalex/book-recipe/sass-folder/styles.scss:10]:
```suggestion
  background-color: $beige-1;
```
No uses `""` para las variables

File: notevenalex/book-recipe/sass-folder/styles.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
+@import "../variables.scss";
```
Comment (DanteCely) [inline notevenalex/book-recipe/sass-folder/styles.scss:1]: Reorganiza las variables, reset y demás archivos sass en una carpeta sass

File: notevenalex/book-recipe/sass-folder/styles.scss [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+h1 {
+  font-family: 'Satisfy', cursive;
+}
+
+.recipe-property-container {
+  width: 60%;
+  height: 90px;
+  background-color: var(--red);
```
Comment (DanteCely) [inline notevenalex/book-recipe/sass-folder/styles.scss:54]: Variable tiene que ser variable sass

**Review by DanteCely — CHANGES_REQUESTED** [review]

**Pull request merged**